force-app/tests/Data Recipes/DynamicSOQLRecipes_Tests.cls

```diff
@@ -13,7 +13,10 @@ private inherited sharing class DynamicSOQLRecipes_Tests {
         List<Account> results = DynamicSOQLRecipes.simpleDynamicSOQLQuery();
         Test.stopTest();
 
-        System.assert(results.size() == 1, 'Expected to find a single account');
+        System.Assert.isTrue(
```

Comment:
Change to `areEqual`

Reply:
Same remark applies to the other tests below.

Reply:
Fixed!